src/store/authentication/saga.ts

```diff
@@ -82,6 +83,11 @@ function* setUserAndChatAccessToken(params: {
         isLoading,
       })
     ),
+    put({
+      type: user
+        ? ChannelsListSagaActionTypes.StartChannelsAndConversationsAutoRefresh
+        : ChannelsListSagaActionTypes.StopChannelsAndConversationsAutoRefresh,
```

Comment:
Should these be separate things rather than the same one? I'm thinking...even if your not authorized you'd still want the channels to be updating because you can see the public ones and you'd want any new ones to show up.

Secondary question, does this mean we'll poll for channels even if the channel app isn't open? Do we care?

Reply:
Do the unread counts update?